tests/Makefile

```diff
@@ -34,7 +34,7 @@ install-deps:
 install-nebula-py: install-deps
 	git clone --branch master https://github.com/vesoft-inc/nebula-python $(CURR_DIR)/nebula-python
 	cd $(CURR_DIR)/nebula-python \
-		&& pip3 install --user . -i $(PYPI_MIRROR) --upgrade
+		&& pip3 install --user --force-reinstall . -i $(PYPI_MIRROR)
```

Comment:
so we need reinstall nebula python each time?

Reply:
I think so. Some runners will keep the previous nebula-python, resulting in a version mismatch.

Reply:
github runner would cache $HOME, and we install python package with `--user` option...

Reply:
yeah.